2주차/11047.py

```diff
@@ -0,0 +1,13 @@
+# 11047 동전 0
+
+n, k = map(int, input().split())
+coin = [int(input()) for _ in range(n)]
+cnt=0
+
+coin.sort(reverse=True)
```

Comment:
```suggestion
coin.reverse()
```
동전이 오름차순으로 주어진다는 조건이 이미 있었으니 굳이 정렬은 안해도 돼요